src/main/java/com/tutorhub/service/ContentS3Service.java

```diff
@@ -0,0 +1,9 @@
+package com.tutorhub.service;
+
+import org.springframework.web.multipart.MultipartFile;
+
+public interface ContentS3Service {
```

Comment:
Maybe you can name it more understandable, like just `S3Service` or `StorageService`

Reply:
Fixed